docs/DeveloperGuide.md

```diff
@@ -233,25 +233,35 @@ Step 5. After the project gets updated, `Model#saveProjectsFolder` is called to
 
 ### Add Event to Project Command [Implemented in v1.2]
```

Comment:
This title can change to

Add Event Feature.

Can help to change Rouchen's Update command to Update Feature. To be consistent.

Then your section on Add command please move it below View Projects Feature and above the Update Feature.